lighthouse-core/lib/i18n/locales/ar.json

```diff
@@ -2060,6 +2069,9 @@
   "lighthouse-core/report/html/renderer/util.js | runtimeUnknown": {
     "message": "غير معروف"
   },
+  "lighthouse-core/report/html/renderer/util.js | showRelevantAudits": {
+    "message": "Show audits relevant to:"
```

Comment:
uhhhh?

Reply:
doesn't look good but I think that setup with that string is already rough for rtl i18n (anything ending with a colon and saying "this stuff here"), maybe that's why it was skipped?

Reply:
Don't think it was skipped for that reason, or at least others weren't (ca/Catalan is not rtl).

Reply:
yeah I found all the util.js german ones were just skipped.